src/components/TodoForm/styles.module.sass

```diff
@@ -1,9 +1,28 @@
 .input
     box-sizing: border-box
-    width: 100%
-    padding: 20px 30px
+    width: 85%
+    padding: 10px 20px
     font-size: 2rem
     margin-bottom: 25px
     border: 1px solid #888
     outline: 0
 
+
+
+
+@mixin button()
```

Comment:
Reusing code by copying is a bad practice. create a separate mixin file and import it if necessary